### ISSUE TITLE
JQuery - Perfect scrollbar - always visible don't work in iframe

### ISSUE THREAD

**Issue by Lohik** [noraesae/perfect-scrollbar#451]
Hi, 

I am creating multistep form (3 steps, so in the beginning are 2nd and 3rd step hidden by display:none).

On third step I have iframe with business conditions and I am using jQuery plugin "Perfect scrollbar" for scrolling iframe content. Scrollbar works great, but I want to make him always visible.

Although I change his opacity in CSS file, when I come to this third step, scrollbar is not visible and appears only after I try to move inside iframe by mousewheel. Then scrollbar appears and stays in windows, so after move with mousewheel it works perfect.

Is there any chance to make scrollbar appear right after I come to third step of form?

Thank you for your answer:)

**Comment by DanielApt** [comment]
[Here's](https://github.com/noraesae/perfect-scrollbar#theme) the documentation on creating a theme for your scrollbar. Simply change `$ps-rail-default-opacity` to `1`, either by overwriting the default, or by creating a new theme.

Also you might need to call `$('.container').perfectScrollbar('update') in advance.

**Comment by Lohik** [comment]
Thank you for your answer, but please tell me more - where I can find $ps-rail-default-opacity ? I changed all opacities in prefect-scrollbar.css and perfect-scrollbar.min.css, but it doesn´t help...

Maybe it would help $('.selector').perfectScrollbar('update') function, may you be please more specific about it? If I should insert it on page with form and iframe or on page with content of iframe? And to what selector  I should "add" it? Sorry, I am really new to jQuery and Javascript...

On page, where is form and iframe I have:

```
<iframe src="/business-conditions" id="iframe" scrolling="no">
</iframe>

<script>
      var iframe = document.querySelector('#iframe');
      iframe.onload = function () {
        Ps.initialize(iframe.contentDocument.querySelector('#Default'), {minScrollbarLength: 50})
		
      };
</script>
```


And in iframe content:

```
<!DOCTYPE html>
<html>
  <head>
    <meta http-equiv="content-type" content="text/html; charset=utf-8">

      <link href="/css/perfect-scrollbar.css" rel="stylesheet">
      <style>
        .contentHolder { position:relative; margin:0px auto; padding:0px; width: 1110px; height: 180px;}
        .contentHolder .content {width: 1080px;}
        .spacer { text-align:center }
      </style>
  </head>
  <body>
    <div id="Default" class="contentHolder">
      <div class="content">
...some <p>....
            </div>
        </div>

</body>
</html>
```

**Comment by noraesae** [comment]
Sorry for the late response.

If you set all `opacity`s, then it should show. If it's not showing, it may be a content and script problem, rather than one of style.

It's really difficult to find the reason without any reproducing case. Can you provide a sample page or jsfiddle for it?

Thanks,

**Comment by noraesae** [comment]
Closed for the inactivity. Please feel free to reopen, or open a new issue. Thanks!
